src/connectors/dynamic-widgets/__tests__/connectDynamicWidgets-test.ts

```diff
@@ -590,6 +590,52 @@ describe('connectDynamicWidgets', () => {
         widgetParams,
       });
     });
+
+    it('warns when facets is unset and more than 20 items are returned from attributesToDisplay', () => {
+      const renderFn = jest.fn();
+      const dynamicWidgets = connectDynamicWidgets(renderFn)({
```

Comment:
consistent with the other tests, but indeed doesn't need to be a mock

Reply:
Got it. Not blocking but I guess we could start doing these adjustments even if it creates inconsistency, otherwise we'll likely never do it. The immediate advantage would be more readable tests.

Up to you.